jOOQ/src/main/java/org/jooq/impl/DSL.java

```diff
@@ -7858,6 +7860,22 @@ public static DropIndexOnStep dropIndexIfExists(Index index) {
     // XXX Access control
     // -------------------------------------------------------------------------
 
+    //todo add javadoc
+    public static Grant<Record> grant(String privilege) {
+        return using(new DefaultConfiguration()).grant(new PrivilegeImpl(name(privilege)));
```

Comment:
For higher API cohesion, it is usually preferrable not to call this constructor here, but to reuse the static method that delegates to the constructor. This may lead to higher stack traces, but also to more quality and better code coverage by the integration tests.